client/me/security-2fa-sms-settings/index.jsx

```diff
@@ -220,46 +202,59 @@ const Security2faSMSSettings = createReactClass( {
 					<FormButtonsBar className="security-2fa-sms-settings__buttons">
 						<FormButton
 							disabled={ this.getSubmitDisabled() }
-							onClick={ function ( event ) {
+							onClick={ ( event ) => {
 								gaRecordEvent( 'Me', 'Clicked On 2fa Use App Button' );
 								this.onVerifyByApp( event );
-							}.bind( this ) }
+							} }
 						>
-							{ this.state.submittingForm ? savingLabel : this.props.translate( 'Verify via App' ) }
+							{ this.props.isUpdatingUserSettings
+								? savingLabel
+								: this.props.translate( 'Verify via App' ) }
 						</FormButton>
 
 						<FormButton
 							disabled={ this.getSubmitDisabled() }
 							isPrimary={ false }
-							onClick={ function ( event ) {
+							onClick={ ( event ) => {
```

Comment:
I wonder if we should declare this as a component method so we won't have to redeclare them on every render.